.prettierrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   jsxBracketSameLine: true,
   trailingComma: 'es5',
   printWidth: 80,
-  parser: 'babylon',
+  parser: 'babel',
```

Comment:
IMO, you should omit the parser and just let it figure it out automatically.

Reply:
How would it automatically figure out the parser without this? 

> Prettier automatically infers the parser from the input file path, so you shouldn't have to change this setting.

https://prettier.io/docs/en/options.html#parser

Reply:
You don't need to define `parser`. Prettier will automatically change the `parser` based on the file type and extension.

Reply:
I suppose it doesn't hurt too bad to have it explicitly stated here then.

Reply:
Do note that it will break if Prettier is ever used on more than just `.js` files.

Reply:
@threepointone use `babel-flow` here to let it handle callable generics properly: https://tinyurl.com/y2o3mpvu